sdkproject/Assets/Mapbox/Unity/Map/IMap.cs

```diff
@@ -11,7 +11,9 @@ public interface IMapReadable
 		Vector2d CenterMercator { get; }
 		float WorldRelativeScale { get; }
 		Vector2d CenterLatitudeLongitude { get; }
-		int Zoom { get; }
+		float Zoom { get; }
+		int InitialZoom { get; }
+		int AbsoluteZoom { get; }
```

Comment:
Please add some comments on what's the difference between these three to make it easier for people learning the SDK.
Also why 2 are `int` and why 1 is `float`.